src/components/results/resultsComponents/ResultsBody.js

```diff
@@ -1,15 +1,49 @@
 import React, { Component } from 'react';
 
 export default class ResultsBody extends Component {
-
+	// sayHello = () => {
+	// 	console.log('clicked')
+	// }
+	// constructor() {
+	// 	super() 
+	// 	this.state = {
+	// 		selectedItems: []
+	// 	}
+	// }
+	// selectItem(key) {
+	// 	console.log(key)
+	// 	//const selectedItems = this.state.selectedItems
+	// 	const { selectedItems } =  this.state
+	// 	const index = selectedItems.indexOf(key)
+	// 	if(index > -1) {
+	// 		selectedItems.splice(index, 1)
+	// 	}else {
+	// 		selectedItems.push(key)
+	// 	}
+	// 	this.setState({
+	// 		selectedItems
+	// 	})
+	// }
+	getRowColor = (key) => {
```

Comment:
I like this idea! I think it makes table rows much more readable 👍 

However, you can actually do this in CSS and it's a bit better for performance. Because we're using bootstrap you can just add the `.table-striped` class to the table. Here's the relevant bit in the documentation: https://getbootstrap.com/docs/3.3/css/#tables-striped. Note the bit that mentions the `nth-child` selector - this is how bootstrap does this in CSS